scripts/west_commands/runners/jlink.py

```diff
@@ -15,13 +16,17 @@
 DEFAULT_JLINK_EXE = 'JLink.exe' if sys.platform == 'win32' else 'JLinkExe'
 DEFAULT_JLINK_GDB_PORT = 2331
 
+class ToggleAction(argparse.Action):
+
+    def __call__(self, parser, args, ignored, option):
+        setattr(args, self.dest, not option.startswith('--no-'))
 
 class JLinkBinaryRunner(ZephyrBinaryRunner):
     '''Runner front-end for the J-Link GDB server.'''
 
     def __init__(self, cfg, device,
                  commander=DEFAULT_JLINK_EXE,
-                 flash_addr=0x0, erase=True,
+                 flash_addr=0x0, erase=True, gen_reset_after_load=False,
```

Comment:
Why the `gen` prefix? Would calling this `reset_after_load` be ok with you?

Reply:
I was following what you did in `sign.py`. I'm fine dropping the `gen` prefix.

Reply:
Done. Please revisit.